aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -1090,6 +1090,18 @@ Tensor zeros(IntArrayRef size,
   return result.zero_();
 }
 
+// TODO: Seems pretty bad that we have to have this separate function.
```

Comment:
How come you can't have old zeros call this?

Reply:
Good point, will do.